lua/lspconfig/arduino_language_server.lua

```diff
@@ -0,0 +1,54 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.arduino_language_server ={
+  default_config = {
+    cmd = {"arduino-language-server"};
+    filetypes = {"arduino"};
+    root_dir = util.root_pattern("*.ino");
```

Comment:
Can you call back to util.path.dirname?

Reply:
I added in the call, following other LSP configs as an example. Let me know if it looks right. I can confirm that the LSP is still functioning properly on my local test arduino project.